cleanup the README file (#225)
kyma-project/serverless-manager: docs/contributor/04-10-testing-strategy.md

# Testing Strategy

Each pull request to the repository triggers CI/CD jobs that verify the Serverless Operator reconciliation logic and run integration tests of the Serverless module.

- `pre-serverless-operator-operator-build` - Compiling the Serverless Operator code and pushing its docker image.

> - `pre-serverless-operator-operator-tests` - Testing the Serverless Operator reconciliation code (Serverless CR CRUD operations).

Reviewer: ```suggestion
- `pre-serverless-operator-operator-tests` - Tests the Serverless Operator reconciliation code (Serverless CR CRUD operations).
```

> - `pre-main-serverless-operator-verify` - Integration testing for the Serverless module installed by Serverless Operator (not using Lifecycle Manager).

Reviewer: ```suggestion
- `pre-main-serverless-operator-verify` - Performs integration testing for the Serverless module installed by Serverless Operator (not using Lifecycle Manager).
```

> - `pull-serverless-module-build` - Bundling a module template manifest that allows testing it against Lifecycle Manager manually. 

Reviewer: ```suggestion
- `pull-serverless-module-build` - Bundles a ModuleTemplate manifest that allows testing it against Lifecycle Manager manually. 
```

Reviewer: If that's the same [ModuleTemplate](https://github.com/kyma-project/lifecycle-manager/blob/main/README.md#basic-concepts) that we mention in Lifecycle Manager docs, I think we should use upper camel case

After the pull request is merged, the following CI/CD jobs are executed:

 - `post-main-serverless-operator-verify` - Installs the Serverless module (using Lifecycle Manager) and runs integration tests of Serverless.

>  - `post-serverless-operator-build` - rebuilds the Serverless module and the module template manifest file that can be submitted to modular Kyma.

Reviewer: ```suggestion
 - `post-serverless-operator-build` - Rebuilds the Serverless module and the ModuleTemplate manifest file that can be submitted to modular Kyma.
```
